util/chunk/chunk.go

```diff
@@ -160,6 +160,41 @@ func (c *Chunk) AppendPartialRow(colIdx int, row Row) {
 	}
 }
 
+// ShadowPartialRowOne use shadow copy to instead of AppendPartialRow,
```

Comment:
1. s/ ShadowPartialRowOne / ShadowCopyPartialRow
2. shadow copies the data of `row` to the first row of `dst`.